src/operations/create_collection.ts

```diff
@@ -91,6 +91,11 @@ export interface CreateCollectionOptions extends CommandOperationOptions {
   expireAfterSeconds?: number;
   /** @experimental */
   encryptedFields?: Document;
+  /**
+   * If set, enables pre-update and post-update document events to be included for any
+   * change streams that listen on this collection.
+   */
+  changeStreamPreAndPostImages?: { enabled: true };
```

Comment:
up for debate here but since sending `{ enabled: false }` is the same as sending nothing, I restricted the type to only enable this option.

Reply:
I think this is fine since it's an opt-in feature. Users will probably only know about it if they want to use it so will probably only ever want to set it to `true`.

Reply:
from slack: I think boolean is a good future proofing choice